website/docs/reference/api/legacy/unleash/admin/features-v2.md

```diff
@@ -609,9 +600,17 @@ Transfer-Encoding: chunked
 
 ## Feature Variants
 
+**Important notes:** 
+1. Variants were previously linked to feature toggles making it impossible to have different variants per environment. We are migrating to variants per environment which will allow you to manage variants independently in each environment. 
+2. As part of the migration, existing variants will be copied to all the environments of its feature which will respect the behaviour prior to having variants per environment.
+3. Clients and SDKs are not affected by this changes as they always access features within one environment.
+4. The export format will change to have variants linked to environments. Old exported formats will be gracefully handled if used to import data.
+5. Because this change adds more data, it won't be possible to down migrate without data loss (i.e. if variants are different per environment, there's no way of representing that in the model where variants were linked to features and applying to all environments)
```

Comment:
This is great information, but feels like more of a blog post than reference documentation, I think. We could rewrite it to a section about changes from 4.x to 4.x+1 or 5 or whatever. What do you think about that?

Reply:
Yes, I agree, it could be part of the release notes or something :thinking: